### PR TITLE
Fix tests php 8.0.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

- [x] add workflok benchmark.

### CODE REVIEW

**Comment by samdark** [comment]
Running benchmark in CI is awesome idea!

File: composer.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
         "irc": "irc://irc.freenode.net/yii",
         "source": "https://github.com/yiisoft/di"
     },
+    "minimum-stability": "dev",
```
Comment (samdark) [inline composer.json:23]: Is that required for phpunit?

Comment (terabytesoftw) [reply composer.json:23]: yes.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**